packages/widget.api.unstable/src/messages/hostMessages.ts

```diff
@@ -30,27 +30,88 @@ export namespace HostMessage {
     export interface UpdateParameters<C extends WidgetConfig<C["parameters"]>> {
       parameters: AsyncParameterValueMap<C>;
     }
+
+    /**
+     * Temporary fetch proxy response
+     * Will be removed in favor of server side proxy
+     */
+    export interface _unstable_FetchResponseSuccess {
```

Comment:
i prefixed these with `_unstable_` because i don't think they should be long lived